doc/bucket-class-crd.md

```diff
@@ -72,3 +78,248 @@ status:
       lastTime: "2019-06-04T13:05:35.473Z"
       troubleshooting: "https://github.com/noobaa/noobaa-core/wiki/Bucket-class-finalizer-troubleshooting"
 ```
+
+# Example
```

Comment:
Minor suggestion:

The comments above for the backing-store-crd.md are also applicable below.